test/test_config.py

```diff
@@ -0,0 +1,31 @@
+"""Tests related to ansiblelint.config module."""
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+import pytest
+
+
+@pytest.mark.parametrize(
+    ("isolated"),
+    (False, True),
+    ids=("normal", "isolated"),
+)
+def test_call_from_outside_venv(isolated: bool) -> None:
+    """Asserts ability to be called without venv activation."""
+    env = None
+    if not isolated:
+        env = {"HOME": Path.home()}
+    py_path = os.path.dirname(sys.executable)
+    # Passing custom env prevents the process from inheriting PATH or other
+    # environment variables from the current process, so we emulate being
+    # called from outside the venv.
+    proc = subprocess.run(
+        [f"{py_path}/ansible-lint", "--version"],
+        check=True,
+        capture_output=True,
+        universal_newlines=True,
+        env=env,
+    )  # type: ignore # env confuses mypy
+    assert ("PATH altered to include" in proc.stderr) == isolated
```

Comment:
For bool/None types, an identity check is recommended:
```suggestion
    warning_emitted = "PATH altered to include" in proc.stderr
    assert warning_emitted is isolated
```